src/wechaty/accessory.py

```diff
@@ -45,18 +49,70 @@ class Accessory:
     _puppet: Optional[Puppet] = None
     _wechaty: Optional[Wechaty] = None
 
+    abstract: bool = True
+
+    def __init__(self):
+        if self.abstract:
+            raise WechatyAccessoryBindingError(
+                'Do not instantiate class {cls} directly, sse with bot.{cls} instead. '
+                'See https://github.com/Chatie/wechaty/issues/1217'.format(
+                    cls=type(self).__name__
+                )
+            )
```

Comment:
`abstract` static attribute is to prevent instantiate `Accessory` subclass, eg: `Room`, directly ?  
We can check the `self` variable type class info to do it.  How do you think about it ?

Reply:
In my opinion, there are two main consideration about this issue:

1. The Accessory class is not desgin to directly expose to user, we only use the class to instanciate a subclass in our `Wechaty.init_puppet()` method, so this design should not affect our users.
2. Checking type and controll logic flow base on it is anyway a less Pythonic way, and checking a type amount Generic types is even worse to understand.

So I decide to change the instantiation lock as it is now.